### PR TITLE
handle error when parse Quantity

### PR DESCRIPTION
**Pull request by nayihz**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
error has been handled int L277. So there is no need to do it again in L280.
https://github.com/kubernetes/kubernetes/blob/a73f45dd96d01bbb2e8b2071a3aafdcdaeed0582/staging/src/k8s.io/kubectl/pkg/cmd/get/sorter.go#L277-L283
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/get/sorter.go [hunk 1]
```diff
@@ -278,9 +278,6 @@ func isLess(i, j reflect.Value) (bool, error) {
 					return sortorder.NaturalLess(itype, jtype), nil
 				}
 				jtypeQuantity, _ := resource.ParseQuantity(jtype)
```
Comment (lauchokyip) [inline staging/src/k8s.io/kubectl/pkg/cmd/get/sorter.go:280]: Thanks for catching this! I wanted to catch the `err` but I forgot to add this. Can you change it to 
```suggestion
				jtypeQuantity, err := resource.ParseQuantity(jtype)
```

instead? Thanks

**Review by lauchokyip — CHANGES_REQUESTED** [review]

File: staging/src/k8s.io/kubectl/pkg/cmd/get/sorter.go (hunk 1 above)
Comment (cmssczy) [reply staging/src/k8s.io/kubectl/pkg/cmd/get/sorter.go]: Done.

**Review by eddiezane — APPROVED** [review]
Thanks!

/approve

**Comment by eddiezane** [comment]
@lauchokyip can you please follow this up with a unit test when you get a chance?

**Pull request merged**